### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by bwbohl**
* fix capitalisation of:
** "Core Contributors"
** "Technical Team"

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,11 +4,11 @@ This document sets out the methods and practices for contributing to the Music E
...
 
 Membership in the Technical Team is informal. Being considered a "member" of this group means you have introduced yourself to the community, have expressed an interest in helping grow and develop MEI, and have demonstrated an ability to contribute to the development of MEI. These contributions may be to the core schema, documentation, tools, tests, or sample encodings. Technical Team members will also help triage issues and help review pull request proposals.
 
-The Technical Team co-leaders will, from time to time, designate and appoint senior members of the technical team to act as Core Contributors. These core contributors will have the responsibility of commit access to the core MEI Schema, and will ensure these contributions are consistent with the spirit and design of the MEI schema.
+The Technical Team Co-chairs will, from time to time, designate and appoint senior members of the Technical Team to act as Core Contributors. These Core Contributors will have the responsibility of commit access to the core MEI Schema, and will ensure these contributions are consistent with the spirit and design of the MEI schema.
 
 ## How to contribute
```
Comment (musicEnfanthen) [inline CONTRIBUTING.md]:
```suggestion
## How to Contribute
```
(other headings are capitalized; please update also heading in line 24: "... for a Contribution")

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -92,18 +92,18 @@ If your commit messages do not contain enough information for a reviewer to unde
 
 ## Responsibilities of Core Contributors
 
-Core contributors (i.e., those with commit access to the MEI schema) are responsible for maintaining and reviewing contributions to the schema. In general, there are four classes of contributions that core contributors need to deal with:
+Core Contributors (i.e., those with commit access to the MEI schema) are responsible for maintaining and reviewing contributions to the schema. In general, there are four classes of contributions that Core Contributors need to deal with:
 
  1. Typos (i.e., Mistakes that affect the documentation, but not the schema).
```
Comment (musicEnfanthen) [inline CONTRIBUTING.md]:
```suggestion
 1. Typos (i.e., mistakes that affect the documentation, but not the schema).
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -92,18 +92,18 @@ If your commit messages do not contain enough information for a reviewer to unde
...
  2. Bug fixes (i.e., Undesired behaviour of the schema)
```
Comment (musicEnfanthen) [inline CONTRIBUTING.md]:
```suggestion
 2. Bug fixes (i.e., undesired behaviour of the schema)
```

**Review by musicEnfanthen — CHANGES_REQUESTED** [review]
Looks good, thank you! Some minor comments included.

File: CONTRIBUTING.md (hunk 1 above)
Comment (musicEnfanthen) [reply CONTRIBUTING.md]: @bwbohl Please update also the heading in line 24: "... for a Contribution". It is "too far away" from the lines affected by this PR, so it is not possible to make an automatic suggestion here.

**Comment by bwbohl** [comment]
just spotted an outdated section on TravisCI thus converting to draft

**Comment by bwbohl** [comment]
> just spotted an outdated section on TravisCI thus converting to draft

can't find what I was wondering about with that, please proceed with reviewing

**Review by rettinghaus — APPROVED** [review]

**Comment by bwbohl** [comment]
found it rephrased it to match the current GitHub Actions CI.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -92,26 +92,28 @@ If your commit messages do not contain enough information for a reviewer to unde
...
+Above all, however, Core Contributors are asked to exercise wisdom and common sense in merging contributions. If they are unsure of the impact of a proposed change, or the documentation of the contribution (pull request messages, commit messages, discussions) are not answered to their satisfaction, they should err on the side of caution and not merge until they are completely satisfied, or call in help from other members of the community.
 
 ### Working with Pull Requests Locally
 
 Sometimes it's easiest if you can check out a pull request's changes locally and run it on your own machine. To do this, you can follow [these instructions](https://gist.github.com/piscisaureus/3342247). This will allow you to further test proposed modifications.
 
-### TravisCI
+### GithubActions
```
Comment (bwbohl) [inline CONTRIBUTING.md]: missing space

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -92,26 +92,28 @@ If your commit messages do not contain enough information for a reviewer to unde
...
 
 ### Working with Pull Requests Locally
 
 Sometimes it's easiest if you can check out a pull request's changes locally and run it on your own machine. To do this, you can follow [these instructions](https://gist.github.com/piscisaureus/3342247). This will allow you to further test proposed modifications.
 
-### TravisCI
+### Github Actions
 
-When a new pull request comes in, the TravisCI service will automatically build the schema using the TEIC Stylesheets, and then validate the test files and the sample encodings against the built schemas. If, for some reason, a change breaks the schema build, or breaks validation, TravisCI will report an error on the pull request. It is then up to the original contributor to fix the issue and submit the fix to the pull request.
+When there is a new push to the develop branch GitHub Actions will automaticall build the MEI schema files and guidelines and publish them at: https://github.com/music-encoding/schema/tree/main/dev
```
Comment (musicEnfanthen) [inline CONTRIBUTING.md]:
```suggestion
When there is a new push to the develop branch, GitHub Actions will automatically build the MEI schema files and guidelines and publish them at: https://github.com/music-encoding/schema/tree/main/dev
```

Comment (musicEnfanthen) [reply CONTRIBUTING.md]: Also: ... a Github Actions script will auto... (?)

**Review by musicEnfanthen — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -92,26 +92,28 @@ If your commit messages do not contain enough information for a reviewer to unde
...
 
 ### Working with Pull Requests Locally
 
 Sometimes it's easiest if you can check out a pull request's changes locally and run it on your own machine. To do this, you can follow [these instructions](https://gist.github.com/piscisaureus/3342247). This will allow you to further test proposed modifications.
 
-### TravisCI
+### Github Actions
 
-When a new pull request comes in, the TravisCI service will automatically build the schema using the TEIC Stylesheets, and then validate the test files and the sample encodings against the built schemas. If, for some reason, a change breaks the schema build, or breaks validation, TravisCI will report an error on the pull request. It is then up to the original contributor to fix the issue and submit the fix to the pull request.
+When there is a new push to the develop branch, a GitHub Actions workflow will automatically build the MEI schema files and guidelines and publish them at: https://github.com/music-encoding/schema/tree/main/dev
```
Comment (musicEnfanthen) [inline CONTRIBUTING.md]: Sorry,  missed this before: Is it correct that the guidelines are published at the schema repo? Should we better mention the guidelines repo, too?

Comment (bwbohl) [reply CONTRIBUTING.md]: thx for spotting missed that on :-o

**Review by musicEnfanthen — APPROVED** [review]

**Comment by musicEnfanthen** [comment]
Sorry, thought that I had pushed the merge button :/ Thanks for the update.

**Pull request merged**